subnet/watch.go

```diff
@@ -23,9 +23,10 @@ import (
 
 func WatchLeases(ctx context.Context, sm Manager, network string, receiver chan []Event) {
 	lw := &leaseWatcher{}
+	var cursor interface{}
```

Comment:
The idea is that the cursor is opaque to these "helper methods" and at the interface level. Each subnet.Manager impl is free to implement the cursor as it sees fit.